### PR TITLE
Added average speed and time in history section. 

### LINKED ISSUE #44: Add average time and speed to history section
I'd like to add a section in the history table that shows the average typing speed and average time of all speed tests taken by the user.

 Something like this:
image

I haven't seen any existing issues for this.  I'd like to be assigned this, let me know if this is something I can add for Hacktoberfest.

### PR DESCRIPTION
**Pull request by alexsam29**
This PR is intended to resolve #44.

I added avg time and speed to both history sections in the practice and profile pages.

## Code Changes
- Added new table below existing history table to display averages
- In `showHistory()` added new table row element creation for the avg table.
- Added new variables in `showHistory()` to calculate average times.
- Simplified `innerHTML` string for the existing table with new variables.
- Added additional CSS to match new table with the existing table.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -202,11 +203,23 @@ const showHistory = () => {
...
-            row.innerHTML = `<td>${htmlEncode(userName)}</td><td>${Math.ceil(item.extracted_words_length / (item.timeTaken / 60))}</td><td>${item.timeTaken}</td><td>${item.difficultyLevel}</td>`;
+            let speed = Math.ceil(item.extracted_words_length / (item.timeTaken / 60));
+            let time = item.timeTaken;
+            avgSpeed += speed;
+            avgTime += time;
+            row.innerHTML = `<td>${htmlEncode(userName)}</td><td>${speed}</td><td>${time}</td><td>${item.difficultyLevel}</td>`;
             historyBody.appendChild(row);
         });
+        avgSpeed = (avgSpeed / historyArray.length).toFixed(1);
+        avgTime = (avgTime / historyArray.length).toFixed(3);;
```
Comment (arpitghura) [inline index.js:220]: Remove double occurrence of `;`

File: profile.js [hunk 2]
```diff
@@ -58,12 +59,23 @@ const showHistory = () => {
...
+            let speed = Math.ceil(item.extracted_words_length / (item.timeTaken / 60));
+            let time = item.timeTaken;
+            avgSpeed += speed;
+            avgTime += time;
+            row.innerHTML = `<td>${htmlEncode(userName)}</td><td>${speed}</td><td>${time}</td><td>${item.difficultyLevel}</td>`;
             historyBody.appendChild(row);
         });
-        console.log("I am from show History");
+        avgSpeed = (avgSpeed / historyArray.length).toFixed(1);
+        avgTime = (avgTime / historyArray.length).toFixed(3);;
```
Comment (arpitghura) [inline profile.js:76]: remove double occurrence of semicolon

**Review by arpitghura — CHANGES_REQUESTED** [review]
Do the required change

**Comment by alexsam29** [comment]
> Do the required change

Sorry, I fixed it.

**Review by arpitghura — APPROVED** [review]

**Pull request merged**